site/pages/view-all-disruptions.page.tsx

```diff
@@ -403,23 +444,69 @@ const ViewAllDisruptions = ({ disruptions, services, newDisruptionId }: ViewAllD
                         <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
                     </svg>
                 </Link>
-                <button
-                    className="govuk-button govuk-button--secondary block"
-                    data-module="govuk-button"
-                    onClick={() => {
-                        if (showFilters) {
-                            setShowFilters(false);
-                        } else {
-                            setShowFilters(true);
-                        }
-                    }}
-                >
-                    {showFilters ? "Hide and clear f" : "F"}ilters
-                </button>
+                <div className="flex">
+                    <button
+                        className="govuk-button govuk-button--secondary block"
+                        data-module="govuk-button"
+                        onClick={() => {
+                            if (showFilters) {
+                                setShowFilters(false);
+                            } else {
+                                setShowFilters(true);
+                            }
+                        }}
+                    >
+                        {showFilters ? "Hide " : "Show "}filters
+                    </button>
+                    <button
+                        className="govuk-button govuk-button--secondary ml-2"
+                        data-module="govuk-button"
+                        onClick={() => {
+                            setFilter({ services: [], operators: [] });
+                            setDisruptionsToDisplay(getPageOfDisruptions(currentPage, disruptions));
+                            setClearButtonClicked(true);
+                        }}
+                    >
+                        Clear filters
+                    </button>
+                </div>
             </div>
 
             {showFilters ? (
                 <>
+                    <OperatorSearch
```

Comment:
Tried searching for an existing disruption with consequence operator "Arriva Yorkshire" and no data was displayed